src/FloatingWindows/FLWIntegration.cpp

```diff
@@ -153,6 +153,23 @@ int LuaSetFloatingWindowPosition(lua_State *L) {
     return 0;
 }
 
+
+int LuaSetImguiFloatingWindowFont(lua_State *L) {
+  if (!lua_islightuserdata(L, 1) || !lua_isnumber(L, 2)) {
+    flywithlua::logMsg(logToDevCon, "FlyWithLua Error: Wrong parameters passed to float_wnd_set_imgui_font");
+    FindAndQuarantine (L);
+    flywithlua::LuaIsRunning = false;
+    return 0;
+  }
+
+  FloatingWindow *wnd = (FloatingWindow *) lua_touserdata(L, 1);
+
+  wnd->setFontNumber(lua_tonumber(L, 2));
```

Comment:
For some reason it seems to be setting the font number to the wrong window if more that one is opened.  It looks the same as title or position so not sure what the issue is.